layouts/page/cudf-polars.html

```diff
@@ -54,23 +50,24 @@ <h2 class="mt-4"><i class="fa-light fa-bolt-lightning"></i> Query Optimizer</h2>
   <div class="col">
     <div class="container">
 
-      <h1 class="mb-4"> RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries </h1>
+      <h1 class="mb-4"> The go-to Polars backend</h1>
       <div class="row">
-        <div class="col-md-6 py-3">
-          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Performance improves as <br> data scales </h2>
-          {{ with .Resources.GetMatch "polars-collect.png" }}
-          <img src="{{ .RelPermalink }}">
-          {{ end }}
+        <div class="col">
+          <p class="subheading">RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries.</p>
```

Comment:
Don't need this sentence --> RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries